desktop/src/app.rs

```diff
@@ -440,7 +442,7 @@ impl App {
                     // On X11 (and possibly other platforms), the window size is not updated immediately.
                     // Wait for the window to be resized to the requested size before we start running
                     // the SWF (which can observe the viewport size in "noScale" mode)
```

Comment:
I feel like this comment could use a word or two's worth of updating with this change...?

Reply:
Maybe swap out `Wait for the window to be resized to the requested size...` for `On a successful resize request, wait for...`?

Reply:
Yep, that sounds good!